dotnet/src/Extensions/Extensions.UnitTests/PromptTemplates/Handlebars/Helpers/KernelSystemHelpersTests.cs

```diff
@@ -272,7 +272,6 @@ public async Task ItThrowsExceptionIfMessageDoesNotContainRoleAsync()
     public static TheoryData<object> JsonObjectsToParse => new()
     {
         new { name = "Alice", age = 25 },
-        "{\"name\":\"Alice\",\"age\":25}",
```

Comment:
We should be careful here with serialized objects. Should we replace this case with encoded json for example?

Reply:
Added this back in and fixed the test